### PR TITLE
Re-adding option to install Tiller

### PR DESCRIPTION
**Pull request by SlickNik**
Adding back the optional checkbox to install Tiller, Helm's server
component.

### CODE REVIEW

**Comment by SlickNik** [comment]
Marking as WIP until license review is complete.

**Comment by jordimassaguerpla** [comment]
I guess this is blocked on https://github.com/kubic-project/caasp-container-manifests/pull/117#pullrequestreview-66146178

**Comment by jordimassaguerpla** [comment]
Once this is approved, remember to cherry-pick the commits to a PR against release-2.0 branch.

File: app/models/pillar.rb [hunk 1]
```diff
@@ -59,6 +59,7 @@ def all_pillars
         http_proxy:            "proxy:http",
         https_proxy:           "proxy:https",
         no_proxy:              "proxy:no_proxy",
+        tiller:                "addons:tiller"
```
Comment (ereslibre) [inline app/models/pillar.rb]: This is missing a trailing comma.

**Comment by ereslibre** [comment]
Updated PR

**Comment by ereslibre** [comment]
Merging as discussed.

**Pull request merged**